templates/cloud/cloud-computing.html

```diff
@@ -2,486 +2,702 @@
 
 {% block title %}What is a cloud computing?{% endblock %}
 
-{% block meta_description %}Cloud computing is an on-demand delivery of IT resources over a network. The cloud resource provisioning process is fully automated, leading to increased agility and significant cost savings.{% endblock meta_description %}
+{% block meta_description %}
+  Cloud computing is an on-demand delivery of IT resources over a network. The cloud resource provisioning process is fully automated, leading to increased agility and significant cost savings.
+{% endblock meta_description %}
 
-{% block meta_copydoc %}https://docs.google.com/document/d/1oIggNSXR5sfl0Ab35x2NIL7I858I7QdPCUXeQI8IIPc/edit#{% endblock meta_copydoc %}
+{% block meta_copydoc %}
+  https://docs.google.com/document/d/1oIggNSXR5sfl0Ab35x2NIL7I858I7QdPCUXeQI8IIPc/edit#
```

Comment:
Can you find the correct doc for this page please. This might also involve updating the other metadata items